irods/test/harness/000_install-irods.Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG linux_vsn="ubuntu:18.04"
+FROM ${linux_vsn}
+ARG irods_vsn="4.3.1-0~bionic"
+
+ARG createdb=""
+ENV ICAT_DEFERRED_CREATEDB /tmp/icat_deferred_createdb
+
+RUN apt update
+RUN apt install -y sudo git
+WORKDIR /root
+RUN git clone http://github.com/d-w-moore/ubuntu_irods_installer
+RUN if [ -n "${createdb}" ]; then \
+        /root/ubuntu_irods_installer/install.sh --w="config-essentials create-db" 0 ;\
+    else \
+        /root/ubuntu_irods_installer/install.sh --w="config-essentials" 0 ;\
+        touch "${ICAT_DEFERRED_CREATEDB}" ;\
+    fi
+RUN /root/ubuntu_irods_installer/install.sh --w="add-package-repo" 0
+RUN /root/ubuntu_irods_installer/install.sh --i=${irods_vsn} -r 4
```

Comment:
these need to be brought local. and, they'll be replace eventually....  by just the minimal irods install commands.